CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-account-http ChangeLog
 
+## 3.0.0 - 2021-03-TBD
+
+### Changed
+- **BREAKING**: Update bedrock-validation in peer and test dependencies.
```

Comment:
this should be something like `Use bedrock-validation@X.  Email addresses must be all lowercase.`

We want to to give the reader some idea why this dep upgrade is a breaking change here without having to dig into changlogs elsewhere if possible.

Reply:
Good catch, totally missed that in my haste.

Reply:
Updated.